packages/rum/src/resourceUtils.ts

```diff
@@ -44,7 +47,7 @@ function formatTiming(start: number, end: number) {
 }
 
 function isValidTiming(start: number, end: number) {
-  return start > 0 && end > 0 && end >= start
+  return start >= 0 && end >= 0 && end >= start
```

Comment:
The 'strictly greater than' is needed here. `entry.redirectStart`, `entry.secureConnectionStart` and associated are equal to `0` when they don't apply (ex: when there is no redirect or secure connection)

Reply:
hum, with initial doc some timings can rightfully be at 0.
I'll look into that and try to add some test to explain that then.